site/pages/api/create-consequence-network.test.ts

```diff
@@ -32,6 +36,43 @@ const defaultNetworkData = {
     disruptionId: defaultDisruptionId,
 };
 
+const defaultDisruptionStartDate = getFutureDateAsString(2);
+const defaultPublishStartDate = getFutureDateAsString(1);
+
+const disruption: FullDisruption = {
+    disruptionId: defaultDisruptionId,
```

Comment:
Could we reuse the disruption from mockData file which also allows us to build a common source of disruption that could be tested across all test cases

Reply:
created a function for it